pkg/kv/kvserver/kvflowcontrol/rac2/token_counter.go

```diff
@@ -452,15 +464,31 @@ func WaitForEval(
 	m := len(scratch)
 	signaledQuorumCount := 0
 
+	var startWait time.Time
+	if traceIndividualDelays {
+		startWait = clock.PhysicalTime()
+	}
+	traceDelayWithReason := func(reason string) {
+		nextStartWait := clock.PhysicalTime()
+		waitDur := nextStartWait.Sub(startWait)
+		startWait = nextStartWait
+		log.Eventf(ctx, "wait-for-eval waited %s for %s", waitDur, reason)
+	}
```

Comment:
Do we need to compute this time? The trace will contain timestamps for each event, which makes this slightly redundant. Consider just logging the events (it's also slightly cheaper, as we'll have fewer calls into the clock).